lib-api/src/main/java/org/ergoplatform/appkit/ErgoProverBuilder.java

```diff
@@ -31,28 +33,22 @@
     ErgoProverBuilder withSecretStorage(SecretStorage storage);
 
     /**
-     * Add DHT prover input using this prover's secret as belonging to the first party (i.e. Alice).
+     * ProveDHTuple is of the form (g, h, u, v) with secret x (and unknown y), where:
+     *   h = g^y
+     *   u = g^x
+     *   v = g^xy
```

Comment:
This is quite cryptic for people like me. 
In order to understand it I had to went through the original example etc.
So I suggest to leave the original naming and add this crypto related information (not replace the original docs)

Reply:
I agree. We should put the original info too.